pkg/connector/capabilities.go

```diff
@@ -63,6 +71,15 @@ func (tc *TwitterClient) GetCapabilities(_ context.Context, _ *bridgev2.Portal)
 				MaxCaptionLength: MaxTextLength,
 				MaxSize:          15 * 1024 * 1024,
 			},
+			event.CapMsgVoice: {
+				MimeTypes: map[string]event.CapabilitySupportLevel{
+					"audio/aac": supportedIfFFmpeg(),
+					"audio/ogg": supportedIfFFmpeg(),
```

Comment:
There should be a format that is always supported, which I guess is `video/mp4` in this case. If it requires some non-standard codec, it can be signaled with `video/mp4; codecs=foo` (or something like that)